lib/sisimai/bite/email/facebook.rb

```diff
@@ -0,0 +1,274 @@
+          'POL-P6', # The message contains a url that has been blocked by Facebook.
+          'POL-P7', # The message does not comply with Facebook's abuse policies and will not be accepted.
+        ],
+        securityerror: [
+          'POL-P1', # Your mail server's IP Address is listed on the Spamhaus PBL.
+          'POL-P2', # Facebook will no longer accept mail from your mail server's IP Address.
+          'POL-P5', # The message contains a virus.
+          'POL-P7', # The message does not comply with Facebook's Domain Authentication requirements.
+        ],
+        notaccept: [
+          'POL-P3', # Facebook is not accepting messages from your mail server. This will persist for 4 to 8 hours.
+          'POL-P4', # Facebook is not accepting messages from your mail server. This will persist for 24 to 48 hours.
+          'POL-T1', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 1 to 2 hours.
+          'POL-T2', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 4 to 8 hours.
+          'POL-T3', # Facebook is not accepting messages from your mail server, but they may be retried later. This will persist for 24 to 48 hours.
+        ],
+        rejected: [
+          'DNS-P1', # Your SMTP MAIL FROM domain does not exist.
+          'DNS-P2', # Your SMTP MAIL FROM domain does not have an MX record.
+          'DNS-T1', # Your SMTP MAIL FROM domain exists but does not currently resolve.
+          'DNS-P3', # Your mail server does not have a reverse DNS record.
+          'DNS-T2', # You mail server's reverse DNS record does not currently resolve.
+        ],
+        systemerror: [
+          'CON-T1', # Facebook's mail server currently has too many connections open to allow another one.
+        ],
+        toomanyconn: [
+          'CON-T3', # Your mail server has opened too many new connections to Facebook's mail servers in a short period of time.
+        ],
+        suspend: [
+          'RCP-T4', # The attempted recipient address is currently deactivated. The user may or may not reactivate it.
+        ],
+        undefined: [
+          'RCP-T1', # The attempted recipient address is not currently available due to an internal system issue. This is a temporary condition.
+          'MSG-T1', # The number of recipients on the message exceeds Facebook's allowed maximum.
+          'CON-T2', # Your mail server currently has too many connections open to Facebook's mail servers.
+          'CON-T4', # Your mail server has exceeded the maximum number of recipients for its current connection.
+        ],
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Facebook: https://www.facebook.com'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
```

Comment:
__[rubocop]__
* Layout/EmptyLineBetweenDefs: Use empty lines between method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods)
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->